### PR TITLE
Retry on 404 - GetSchemaItems

### PR DESCRIPTION
**Pull request by arik123**

### CODE REVIEW

**Review by idinium96 — APPROVED** [review]
Thank you very much!

File: lib/schema.js [hunk 1]
```diff
@@ -2344,20 +2345,24 @@ function getAllSchemaItems(apiKey, next, items, callback) {
...
+            return statusCode && (statusCode === 429 || statusCode === 404 || (500 <= statusCode && statusCode < 600));
+        }
+    }, (err, response, body) => {
         if (err) {
             return callback(err);
         }
+        result = body?.response;
 
-        items = (items || []).concat(result.items);
+        items = (items || []).concat(result?.items || []);
```
Comment (idinium96) [inline lib/schema.js:2363]: image

Hmmm, I think this pose another problem,

Comment (arik123) [reply lib/schema.js:2363]: that's kinda weird, but if you think it's because of that specific line, let's see what happens if you revert it

Comment (idinium96) [reply lib/schema.js:2363]: I am not sure tbh, but I wonder why it's missing `item_class` in this case.
Not sure if it's still happening at the moment.

**Pull request merged**